meta-facebook/at-mc/prj.conf

```diff
@@ -66,7 +66,7 @@ CONFIG_SHELL_METAKEYS=n
 CONFIG_SHELL_HELP=n
 CONFIG_SHELL_HISTORY=n
 CONFIG_SHELL_STATS=n
-CONFIG_KERNEL_SHELL=n
+CONFIG_KERNEL_SHELL=y
```

Comment:
Do u intend to keep this kernel config, i don't see the PR description mentioning anything about kernel shell?

Reply:
Hi @jagpalgill 
Yes, we would like to enable kernel config until PVT stage. Update the description.